Entities/Structures/Trampoline/TrampolineLogic.as

```diff
@@ -41,12 +42,20 @@ void onTick(CBlob@ this)
 
 	Vec2f ray = holder.getAimPos() - this.getPosition();
 	ray.Normalize();
-
+	
 	f32 angle = ray.Angle();
-	angle = angle > 135 || angle < 45? (holder.isFacingLeft()? 135 : 45) : 90;
-	angle -= 90;
+	CControls@ controls = getControls();
 
-	this.setAngleDegrees(-angle);
+	if (controls.isKeyPressed(KEY_LCONTROL))
+	{
+		angle = angle > 135 || angle < 45? (holder.isFacingLeft()? 135 : 45) : 90;
+		angle -= 90;
+		this.setAngleDegrees(-angle);
+	}
+	else
+	{
+		this.setAngleDegrees(-angle+90);
```

Comment:
The trampoline spazzes out if you move your cursor to the centre point of rotation. There should be a threshold at which it stops updating its angle if the cursor is too close.